contrib/mesos/docs/ha.md

```diff
@@ -1,4 +1,25 @@
-## High Availability
+# High Availability
+
+Kubernetes on Mesos supports 2 HA modes:
+
+* [Hot-standby](#hot-standby)
+* [Cold-standby](#cold-standby)
+
+In the hot-standby mode, all master components (apiserver, controller manager, and scheduler) actively run 
```

Comment:
Also the different modes are referred to as "the hot-standby" and "the cold-standby" here, but later on they're referred to as simply "hot-standby" or "cold-standby". I'd prefer a consistent use of the later form.